src/common/dtpthread.c

```diff
@@ -56,10 +56,10 @@ int dt_pthread_create(pthread_t *thread, void *(*start_routine)(void *), void *a
   if(ret != 0 || stacksize < WANTED_THREADS_STACK_SIZE /*|| 1*/)
   {
     // looks like we need to bump/set it...
-
+#if 0
```

Comment:
Either we keep the code or we remove it. But having code disabled this was without further comment is not good to me. Any dev reading this will ask himself why it is still there.

Maybe this should be triggered only with `DT_DEBUG_DEV`?

Reply:
> Either we keep the code or we remove it. But having code disabled this was without further comment is not good to me. Any dev reading this will ask himself why it is still there.

Yes, you are right, my change proposal was not the best.


> Maybe this should be triggered only with `DT_DEBUG_DEV`?

Even in this case, these messages will not help in debugging, they will always be issued, even in regular situations.
I would remove this code. I've been seeing this message on Windows for ages. Apparently, this is specific behavior of the Windows implementation. So this message does not carry any useful information.

If no one is against it, I will change the PR and remove this code completely.

Reply:
I'm ok with that.